karma.conf.maker.js

```diff
@@ -136,7 +136,7 @@ module.exports = function(codeCoverage, browserstack, watchMode, file) {
     // preprocess matching files before serving them to the browser
     // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
     preprocessors: {
-      'test/test_index.js': ['webpack', 'sourcemap']
+      [file != null ? file : 'test/test_index.js']: ['webpack', 'sourcemap']
```

Comment:
Don't change this one in your PR.  
If you think that this needs to be changed, please open separate PR for it.